related-efforts.md

```diff
@@ -11,22 +11,73 @@ has_toc: false
 
 Software and services that implement the SCITT architecture support a variety of use cases. Below is an inventory of related efforts, be it specifications, implementations, or initiatives, where SCITT is applicable.
 
-## Supply Chain
+## Software Supply Chain Security
+A list of efforts for protecting the security of software supply chain
+
+- [Microsoft SDL](https://www.microsoft.com/en-us/securityengineering/sdl)
+- [Google Software Delivery Shield](https://github.blog/2023-04-19-introducing-npm-package-provenance/)
+- [SigStore](https://www.sigstore.dev/)
+A [blog post](https://openssf.org/case-studies/2024/02/16/scaling-up-supply-chain-security-implementing-sigstore-for-seamless-container-image-signing/)
+describing how SigStore is used by Yahoo!
+
+
+## Artifacts
+A list of artifact types that can be (potentially) recorded in a Transparency registry
 
 ### Bill of Materials
 
 - [CycloneDX](https://cyclonedx.org)
 - [SPDX](https://spdx.dev/)
 - [Software Identificatino Tags - SWID (ISO/IEC 19770-2:2015)](https://www.iso.org/standard/65666.html)
 - [CoSWID - Concise Software Identification Tags (RFC9393)](https://datatracker.ietf.org/doc/rfc9393/)
+- [Microsoft's SBOM tool](https://github.com/microsoft/sbom-tool)
 
-### Digital Signatures
```

Comment:
Is there a reason we are removing OpenPubKey?

Reply:
It is not removed. I created a sub section "Related tools" and inside there another sub-section "Digital Signatures and Key Transparency" and I have moved it there together with some other tools.

Reply:
Oh, woops. Sorry. That makes a ton of sense.